src/routes/RemediationDetailsComponents/helpers.js

```diff
@@ -75,3 +75,15 @@ export const getTimeAgo = (date) => {
   const diffYears = Math.floor(diffDays / 365);
   return `${diffYears} year${diffYears === 1 ? '' : 's'} ago`;
 };
+
+// eslint-disable-next-line react/display-name
+export const renderComponent = (Component, props) => (_data, _id, entity) =>
+  <Component {...entity} {...props} />;
+
+export const chunkArray = (array, size) => {
```

Comment:
I believe using lodash function `chunk` could save some code here: https://www.geeksforgeeks.org/lodash-_-chunk-method/

Reply:
ahh this is nice, added!